tests/testsuite/inheritable_workspace_fields.rs

```diff
@@ -631,8 +638,8 @@ fn inherit_workspace_fields() {
           "features": {},
           "homepage": "https://www.rust-lang.org",
           "keywords": ["cli"],
-          "license": null,
-          "license_file": null,
+          "license": "MIT",
+          "license_file": "[..]LICENSE",
```

Comment:
I feel like we'll get a better idea if the tests are doing what we expect if they were to use `[ROOT]` instead of `[..]`

Reply:
I'll change it to `[ROOT]`, I agree that it gives a better idea of what is going on

Reply:
I tried to do this and it didn't work for some reason. I just changed it for `../LICENSE`

Reply:
Oh right, its relative now.  Thats what I'm looking for, thanks!